Launcher/Classes/ViewControllers/TasksViewController.swift

```diff
@@ -414,6 +414,17 @@ class TasksViewController: NSViewController {
             arguments.append("")
         }
         
+        if let deviceIP = applicationStateHandler.deviceIP,
+            let bundleID = applicationStateHandler.bundleID,
+            physicalDeviceRadioButton.state == .on,
+            !deviceIP.isEmpty {
+            arguments.append("export DEVICE_ENDPOINT=http://\(deviceIP):37265")
```

Comment:
Is this port (the `37265`, I think it's a port?) always the same? Maybe add a constant for it?

Reply:
Yes this will be always the same. It is Calabash default port. I will add it to the constants